onnx2trt_utils.hpp

```diff
@@ -145,6 +145,55 @@ inline nvinfer1::Dims convert_dims(OnnxDims const& onnx_dims) {
   return trt_dims;
 }
 
+inline bool convert_weight_descriptor(onnxTensorDescriptor const &desc,
+                                      onnx2trt::ShapedWeights *weights) {
+  // We only support grabbing weight from CPU memory now
+  assert(desc.memoryType == ONNXIFI_MEMORY_TYPE_CPU);
```

Comment:
This check should be in model parsing, as we shouldn't `abort()` the whole process

Reply:
OK.